### DISCUSSION TITLE
Some movies go into "shows" folder

### DISCUSSION THREAD

**Discussion by Il1kethestars** [itsToggle/plex_debrid#634] [Q&A]
I'm definitely a beginner at this so please bear with me. 

My setup: windows 11, rclone and plex_debrid running fine

Sometimes when I add a movie to my watchlist in Plex it'll do everything as it should except it will put the movie into the "shows" virtual folder. This causes issues with plex and it won't recognize it correctly. I'm not positive about this but it seems that when it happens, those movies have multiple files (extras, outtakes, etc along with the actual movie file) so I'm not sure if that plays a role in this problem happening. Most recently this has happened with The Abyss and Django. 

Thanks for any tips!

**Comment by dariokan82** [comment]
Hey there, 

I am having the same issue - did you ever figure out a solution? Thanks!

**Comment by Il1kethestars** [comment]
Hey fellow debrider,

In a way I did figure out a solution but in a way it's more "avoiding the
problem" than anything.

There may be a correct, or more "direct", solution out there but I haven't
found it. What I've begun doing is a bit more tedious but I haven't had the
aforementioned issues and it's solved other minor annoyances as well.

I have moved away from using the watchlist button as the sole way to get my
movies/shows and I've accepted that I have to be more "hands on" with
adding movies/shows. I'll explain the process that I've found to be most
effective:


   - Start by adding the thing to my watchlist.
   - I'll immediately pull up the realdebrid (X:) drive and see if it's
   added the media to the correct folder.
      - If it grabbed a good version and put it into the correct folder,
      the gods have smiled upon me this day --> remove from watchlist. This
      ensures that the script doesn't continue to look for that media even if I
      have it. Before I started removing things from the watchlist, I noticed
      that sometimes I'd end up with multiple copies of the same
thing, sometimes
      up to 5 copies...I don't know why this would happen.
      - If there are problems I'll have to get it in a more "manual" way.
      Here are most of the biggest problems that I've encountered: media goes
      into the wrong folder (just like you're asking about); it
grabbed "extras"
      and plex gets confused, then puts those extras in place of the
actual media
      (this happens with shows most often); the quality of the media
is absolute
      shit; the format of the media is bad (even though I set up rules in the
      plex_debrid script, sometimes it'll still grab wonky formats), etc...
   - My workflow for getting the media manually:
   - Remove the media from my watchlist, go into real-debrid (I've begun
      using debridmediamanager (dmm), it has a far more user friendly UI) and
      remove the bad version.
      - To find the media manually, I sometimes just go to torrent sites
      and search it up. If there's one that is high quality and has quite a few
      seeds, I'll just copy the magnet into real-debrid/dmm, unselect
everything
      except the video file(s).
* If you aren't familiar with this process, let me know and I can walk you
      through it. *
         - If that version is already cached in rd, I'll open (X:) and be
         sure it went into the correct folder, usually it does, then
I'll refresh my
         plex library and test it.
         - If it's not cached, goes into the wrong folder, or plex hates
         that version, I'll just remove it from rd and delete it from
plex, then
         find a different torrent.
      - An alternative that I've found really slick is to use the search
      function built into qbittorrent. I don't actually download anything with
      qbit, I'll just search for media, right click the one I want, and inspect
      it - which brings me to the torrent website.

This was a lot. If you have any questions or if I wasn't clear on anything,
feel free to let me know and I'd be happy to walk you through my process.

**Reply by dariokan82** [reply]
You're a saint! Thanks for writing this up for me.

Funnily enough, I've started looking into DMM today - do you use it self-hosted or via the website? 

Re: your workflow, it makes sense. I have noticed that PlexDebrid often fudges the TV Shows (in my case, it keeps them in the "default" mount folder; my solution for this has been to add this folder to the "TV Shows" section of Plex). 

But, in the past few weeks, I've noticed some films ending up in the TV Show folder. My solution so far has been downloading the movie I wanted to watch and that was it. But I use a similar workflow as yours for when PlexDebrid doesn't find any media - I just add it manually to Real Debrid and it magically appears.

Fingers crossed it gets solved in a future release, and thanks again for your kind answer!

**Comment by Il1kethestars** [comment]
You're very welcome! This has been quite the learning process and I know
that any guidance can make a big difference.

DMM is so helpful! I just go to the website to use it. I'm not sure what
you mean by using it self-hosted. Maybe that doesn't pertain to me because
I have my plex and plex-debrid stuff on my daily use pc.

I agree, TV shows seem to be the most difficult for everything to get
right. My goal is usually to only resort to downloading things that I love
and really want to actually have a "real" copy of.

I think that the biggest part of this whole process for me was realizing
that it's not going to be perfect and there will always be some fiddling
around and manual corrections. This setup does make a lot of things much
easier and is amazing for preserving disk space but I just accepted that
there will always have to be effort, it's just shifted slightly by using
plex-debrid.

Good luck and happy torrenting!
